app/src/test/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUiRendererTest.kt

```diff
@@ -15,14 +17,10 @@ class CustomDrugEntryUiRendererTest {
   private val uiRenderer = CustomDrugEntryUiRenderer(ui, dosagePlaceholder = "mg")
   private val drugName = "Amlodipine"
   private val dosagePlaceholder = "mg"
-  private val drugFrequencyChoiceList = listOf(
-      DrugFrequencyChoiceItem(drugFrequency = null, labelResId = R.string.custom_drug_entry_sheet_frequency_none),
-      DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.OD, labelResId = R.string.custom_drug_entry_sheet_frequency_OD),
-      DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.BD, labelResId = R.string.custom_drug_entry_sheet_frequency_BD),
-      DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.QDS, labelResId = R.string.custom_drug_entry_sheet_frequency_QDS),
-      DrugFrequencyChoiceItem(drugFrequency = DrugFrequency.TDS, labelResId = R.string.custom_drug_entry_sheet_frequency_TDS))
+  private val drugFrequencyFactory = DrugFrequencyFactory(CommonDrugFrequencyProvider())
+  private val drugFrequencyChoiceItems = DrugFrequencyChoiceItems(drugFrequencyFactory.provideFields()).items
```

Comment:
Same comment as above, since we only need `drugFrequencyFactory.provideFields()`. We don't need to wrap it with `DrugFrequencyChoiceItems` and then get `items`. We can directly use `provideFields()`